kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/cases/CaseRuntimeDataServiceIntegrationTest.java

```diff
@@ -554,6 +554,35 @@ public void testGetCaseInstancesWithData() {
         assertEquals("first case started", instance.getCaseFile().getData().get("s"));
     }
 
+    @Test
+    public void testByPassAuthCaseTasksAssignedAsPotentialOwner() throws Exception {
+        String caseId = startUserTaskCase(USER_YODA, USER_JOHN);
+        assertNotNull(caseId);
+        assertTrue(caseId.startsWith(CASE_HR_ID_PREFIX));
+        try {
+            List<TaskSummary> tasks = caseClient.findCaseTasksAssignedAsPotentialOwner(caseId, USER_YODA, 0, 10);
+            assertNotNull(tasks);
+            assertEquals(1, tasks.size());
+            assertEquals("Hello1", tasks.get(0).getName());
+
+            changeUser(USER_JOHN);
+
+            tasks = caseClient.findCaseTasksAssignedAsPotentialOwner(caseId, USER_MARY, 0, 10);
+            assertNotNull(tasks);
+            assertEquals(0, tasks.size());
+
+            tasks = caseClient.findCaseTasksAssignedAsPotentialOwner(caseId, USER_YODA, 0, 10);
+            assertNotNull(tasks);
+            assertEquals(1, tasks.size());
+            assertEquals("Hello1", tasks.get(0).getName());
+
+        } finally {
+            changeUser(USER_YODA);
```

Comment:
This is not really needed, user is updated at the `after` annotated method in the class

Reply:
If I dont call changeUser(), its not able to destroy case instance,test is failing with exception
`org.kie.server.api.exception.KieServicesException: User john is not authorized to DESTROY_CASE on case HR-0000000199`

Reply:
you're right